### PR TITLE
Added additional border parameters

### PR DESCRIPTION
**Pull request by hemish11**
Ticket: https://github.com/EnsembleUI/ensemble/issues/898

### CODE REVIEW

File: lib/widget/helpers/controllers.dart [hunk 1]
```diff
@@ -239,6 +260,92 @@ class BoxController extends WidgetController {
...
+
+    if (individualBorderColorData.length == 1) {
+      borderColor = convertStringToColor(individualBorderColorData[0]);
+
+      topBorderColor = borderColor;
+      bottomBorderColor = borderColor;
+      leftBorderColor = borderColor;
+      rightBorderColor = borderColor;
+    } else if (individualBorderColorData.length == 2) {
+      topBorderColor = convertStringToColor(individualBorderColorData[0]);
```
Comment (vusters) [inline lib/widget/helpers/controllers.dart:278]: shorten and don't repeat logic to minimize typo errors:
topBorderColor = bottomBorderColor = ..
Same for below.

**Comment by vusters** [comment]
Make sure to update ensemble_schema.json. This is how the user know which properties are available.

**Comment by hemish11** [comment]
@vusters Please re-review the changes

**Comment by hemish11** [comment]
@vusters Should I close this? Or is there anything else left?

**Comment by vusters** [comment]
Please add schema to it.

**Review by vusters — APPROVED** [review]

**Pull request merged**